spek-dsl/jvm/build.gradle

```diff
@@ -0,0 +1,18 @@
+apply from: "$rootDir/gradle/common/dependencies.gradle"
+apply from: "$rootDir/gradle/common/kotlin-jvm.gradle"
+apply plugin: 'org.jetbrains.dokka'
+
+task sourceJar(type: Jar) {
+    from project.sourceSets.main.allSource
+    classifier "sources"
+}
+
```

Comment:
nit: rm empty line :)

Reply:
Let's leave it for now :) Default settings for any IDE I have used always ensures an empty line at the end of the file.

Reply:
ok, but it's not end of file :trollface: